WePool/src/app/profile/survey-components/question-component/question-component.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Does not need to be done this commit, but I'm thinking this file should probably have "test" in the name or be under a test directory or something.